### PR TITLE
Add wait for ajax helper

### PR DESCRIPTION
**Pull request by money**
Reduce test failures when Capybara tries to interact with form that is dependent on an api request by waiting for ajax request(s) complete.

### CODE REVIEW

File: lib/screening.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@ def complete_and_submit
 
   def click_submit
     find('div.page-header-container').click_button('SUBMIT')
+    wait_for_ajax(time: 5)
```
Comment (NagaPradeepGajula-osi) [inline lib/screening.rb]: cool thanks for adding this change.

File: spec/support/matchers/be_before_after.rb [hunk 2]
```diff
@@ -1,13 +0,0 @@
-# frozen_string_literal: true
-
```
Comment (NagaPradeepGajula-osi) [inline spec/support/matchers/be_before_after.rb]: less and unused code is always good.

**Comment by NagaPradeepGajula-osi** [comment]
@brunoOSI @money Are we planning to enable the smoke tests to use the persona based tests?

**Comment by brunoOSI** [comment]
@money for president

**Comment by brunoOSI** [comment]
@NagaPradeepGajula-osi the idea is to add more juice to the smoke test yes.
It should get to a point where we are confident about the integration with other services

**Pull request merged**